### DISCUSSION TITLE
Another PNM code is now public

### DISCUSSION THREAD

**Discussion by TristanAgaesse** [PMEAL/OpenPNM#2561] [Ideas]
Hello,

I recently made public a Pore Network Modeling code that I wrote during my PhD.
Feel free to have a look and use whatever is useful to you.

Some original features:
- model to simulate water condensation in a thin porous medium
- model to simulate the effect of a hydrophobicity change on the water invasion patterns

Here is the code:
https://github.com/TristanAgaesse/PNM_Matlab

**Comment by jgostick** [comment]
Thanks for sharing Tristan!
